config/serverless.oblt.yml

```diff
@@ -16,6 +16,9 @@ xpack.serverless.observability.enabled: true
 ## Set the home route
 uiSettings.overrides.defaultRoute: /app/observability/landing
 
+# Customize empty page state for analytics apps
+no_data_page.analyticsNoDataPageFlavor: 'serverless_observability'
```

Comment:
What's the reason for `snake_case` here?

Reply:
Wow, are those interchangeable? I'll fix